privacy_evaluator/models/torch/dcti.py

```diff
@@ -0,0 +1,77 @@
+import os
+import sys
+
+import torch
+import torch.nn as nn
+
+
+__all__ = ["DCTI", "dcti"]
```

Comment:
Looks all really great! I am just wondering if we already need private classes and functions. So far we don't really know how the interfaces of the classes should look like.

Reply:
The purpose of this line is to make the Block class invisible from the outside, as it is an implementation detail of the model :) 

In general I tried to mimic the public API of `torchvision` models. See e.g. https://github.com/pytorch/vision/blob/master/torchvision/models/resnet.py